code-challenges.test.js

```diff
@@ -54,6 +103,55 @@ var vowelTester2 = "academy"
 var vowelTester3 = "challenges"
 // Expected output: 2
 
+// 3a) Pseudo code: 
+// Create describe, it, and expect statements to describe the problem statement
+// Add appropriate inputs/outputs, with one expect line per input/output pair
+// Run the test on terminal (yarn jest) prior to creating the function to ensure it turns red (note: if set up correctly, should get an error saying the function is not defined)
 
+describe("vowelOfEnmity", () => {
+    it("a function that takes in a string, searches for the first vowel in the string (if there is one), and returns the index of said vowel", () => {
+      expect(vowelOfEnmity(vowelTester1)).toEqual(1)
+      expect(vowelOfEnmity(vowelTester2)).toEqual(0)
+      expect(vowelOfEnmity(vowelTester3)).toEqual(2)
+    })
+  })
 
 // b) Create the function that makes the test pass.
+
+// 3b) Pseudo code: 
+// Declare a function and name it vowelOfEnmity (const declaration)
+// Name the parameter as string
+// Create a variable named vowelArr, and initialize it as an empty array (let declaration)
+// Create several conditional statements, each of which finds the index of one of the vowels via .indexOf(), and if the result isn't -1, will .push each one into variable vowelArr
+// Use Math.min(VowelArr) to find the lowest value within vowelArr and return the result
+
+// const vowelOfEnmity = (string) => {
+//     let vowelArr = []
+//     if(string.indexOf("a") !== -1){
+//         vowelArr.push(string.indexOf("a"))
+//     }
+//     if(string.indexOf("e") !== -1){
+//         vowelArr.push(string.indexOf("e"))
+//     }
+//     if(string.indexOf("i") !== -1){
+//         vowelArr.push(string.indexOf("i"))
+//     }
+//     if(string.indexOf("o") !== -1){
+//         vowelArr.push(string.indexOf("o"))
+//     }
+//     if(string.indexOf("u") !== -1){
+//         vowelArr.push(string.indexOf("u"))
+//     }
+//     return (vowelArr.sort())[0]
+//}
+const vowelOfEnmity = (string) => {
+    let vowelArr = []
+    string.indexOf("a") !== -1 ? vowelArr.push(string.indexOf("a")) : null
+    string.indexOf("e") !== -1 ? vowelArr.push(string.indexOf("e")) : null
+    string.indexOf("i") !== -1 ? vowelArr.push(string.indexOf("i")) : null
+    string.indexOf("o") !== -1 ? vowelArr.push(string.indexOf("o")) : null
+    string.indexOf("u") !== -1 ? vowelArr.push(string.indexOf("u")) : null
+    return (vowelArr.sort())[0]
+}
+
```

Comment:
I am tracking your logic and it is definitely a valid approach. I think looking at some iteration might be a good refactor.